<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alarmItemWidget</class>
 <widget class="QWidget" name="alarmItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>44</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="alarm">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;AlarmName&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="alarmStatus">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;Label, Repeat, Once&gt;</string>
     </property>
     <property name="indent">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QToolButton" name="editAlarm">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="document-edit">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="2">
    <widget class="QToolButton" name="removeAlarm">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="edit-delete">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
